### ISSUE TITLE
Suggestions only given for first word in line

### ISSUE THREAD

**Issue by reagle** [priner/micro-aspell-plugin#5]
If I have two words underlined as mispelled and want to leave the first as is (without adding to the dictionary) the suggestions remain for the even if the cursor is in the second.
<img width="909" alt="Screen Shot 2020-07-30 at 10 09 50 AM" src="https://user-images.githubusercontent.com/596814/88933051-e6380580-d24c-11ea-9f54-48740decaa35.png">

**Comment by reagle** [comment]
Can anyone confirm?

**Comment by mardukbp** [comment]
Yes, this is how it currently works. I think that when the cursor is on a flagged word the status line should show the suggestions for this word. I just proposed that navigating to a flagged word be implemented #8. That should contribute to fixing this issue.
